test/Interop/Cxx/class/method/methods-addressable-silgen.swift

```diff
@@ -0,0 +1,17 @@
+
+// RUN: %target-swift-emit-silgen -I %S/Inputs -enable-experimental-feature AddressableParameters -enable-experimental-cxx-interop %s | %FileCheck %s
+
+import Methods
+
+// REQUIRES: swift_feature_AddressableParameters
+
+public func addressableTest(x: borrowing @_addressable NonTrivialInWrapper) {
+  let m = HasMethods()
+  m.nonTrivialTakesRef(x)
+  // No copy from the argument to the apply.
+  // CHECK: bb0([[INPUT:%[0-9]+]] : @noImplicitCopy $*NonTrivialInWrapper):
```

Comment:
nit: what is `bb0`? Is it part of an identifier somewhere in the test case that I'm missing, or is it a compiler-generated name? If the latter, can you 

Also, it might be a good idea to guard these `CHECK` directives in something that ensures these statements are in the body of the `addressableTest` function. I know it's not necessary for this test/compilation unit but it can only help make the diagnostics more precise if the test does fail.

Reply:
I added a check to make sure we are checking against the body of the right function. I think the naming of `bb0` as the first basic block is here to stay so I did not update that one.